src/JB721TiersHookStore.sol

```diff
@@ -822,16 +835,24 @@ contract JB721TiersHookStore is IJB721TiersHookStore {
             storedTier = _storedTierOf[msg.sender][tierId];
 
             // Parse the flags.
-            (bool allowOwnerMint,,,) = _unpackBools(storedTier.packedBools);
+            (bool allowOwnerMint,,,,) = _unpackBools(storedTier.packedBools);
 
             // If this is an owner mint, make sure owner minting is allowed.
             if (isOwnerMint && !allowOwnerMint) revert CANT_MINT_MANUALLY();
 
             // Make sure the provided tier exists (tiers cannot have a supply of 0).
             if (storedTier.initialSupply == 0) revert INVALID_TIER();
 
+            // Get a reference to the price.
+            uint256 price = storedTier.price;
+
+            // Apply a discount if needed.
+            if (storedTier.discountPercent > 0) {
+                price -= mulDiv(price, storedTier.discountPercent, JB721Constants.MAX_DISCOUNT_PERCENT);
```

Comment:
Doesn't this give half of the intended discountPercent?

Reply:
should be full?

let's say discountPercent is JB721Constants.MAX_DISCOUNT_PERCENT/2, then it goes `price -= price/2` which is half right? 

anyways, a test would be nice. :]